libcudacxx/include/cuda/std/bitset

```diff
@@ -681,13 +679,9 @@ public:
     const _CharT* __str, size_t __n = static_cast<size_t>(-1), _CharT __zero = _CharT('0'), _CharT __one = _CharT('1'))
   {
     size_t __rlen = _CUDA_VSTD::min(__n, char_traits<_CharT>::length(__str));
-#if defined(_LIBCUDACXX_HAS_STRING_VIEW)
-    __init_from_string_view(basic_string_view<_CharT>(__str, __rlen), __zero, __one);
```

Comment:
This was leading to the constexpr evaluator going crazy

I assume it has to do with us calling `operator[]` on an actual type as opposed to just on a plain pointer